src/main/webapp/styles/change-profile.css

```diff
@@ -0,0 +1,75 @@
+#popup-profile h2 {
+  color: black;
+}
+
+#popup-profile p {
+  color: black;
+}
+/* --- Formatting for Popup Box --- */
```

Comment:
Please use the [recommended style](https://google.github.io/styleguide/jsguide.html#formatting-comments) for comments in JavaScript